Dockerfile.js-shopping-cart

```diff
@@ -1,4 +1,4 @@
-FROM node:8.15
+FROM node:12
```

Comment:
While this change has nothing to do with this PR, node 12 is needed by the js shopping cart `package.json`, and I did need to rebuild shopping cart to test the knative support which was what this change was done for.